robot_skills/src/robot_skills/arm/arms.py

```diff
@@ -1,25 +1,35 @@
-from __future__ import print_function
+from __future__ import print_function, annotations
```

Comment:
I think both of these are not needed. `print_function` was needed for python2/3 transition. `annotations` is needed when you want to use a class as type in the class itself.

Reply:
If we remove annotations we must also change line 80 to `def __init__(self, arm: "Arm", ...`. 
`print_function` we can remove

Reply:
Oh I missed that one. I will change it.